lib/raidenclient/RaidenClient.ts

```diff
@@ -83,22 +83,24 @@ class RaidenClient extends BaseClient {
 
     this.port = port;
     this.host = host;
+
     if (disable) {
-      this.setStatus(ClientStatus.DISABLED);
+      return;
     }
+    this.setStatus(ClientStatus.DISCONNETED);
+
+    this.init();
```

Comment:
Here too we shouldn't call the async `init` method from with the synchronous constructor.